src/Codeworx.Rest.AspNetCore/ContractModelProvider.cs

```diff
@@ -68,6 +68,15 @@ public void OnProvidersExecuting(ApplicationModelProviderContext context)
                             }
                         }
                     }
+                    else
+                    {
+                        actionsToRemove.Add(action);
+                    }
+                }
+
+                foreach (var action in actionsToRemove)
```

Comment:
Remove actions only if at least one RestRouteAttribute was found on the controller.